src/main/java/domain/Results.java

```diff
@@ -42,30 +45,40 @@ public String getResult(int position) {
                 .getName();
     }
 
-    public List<String> getFinalResults(List<String> playersName, List<Integer> lastPositions, String targetPlayer) {
-        List<String> results = new ArrayList<>();
+    public Results getFinalResults(List<String> playersName, List<Integer> lastPositions, String targetPlayer) {
+        if (targetPlayer.equals(ALL_PLAYER)) {
```

Comment:
다음과 같은 형태로도 사용하는데 이유가 무엇이고 어떤 점이 좋을까요?
```suggestion
        if (ALL_PLAYER.equals(targetPlayer)) {
```

Reply:
NullPointerException을 방지할 수 있어 실수를 줄일 수 있는 좋은 방법 같아요!!